### ISSUE TITLE
transect setup form - check transect length calculation

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/SPRING#75]
relates to: https://github.com/BiologicalRecordsCentre/ABLE/issues/460

@andrewvanbreda please can you check that we don't have this problem with the eBMS form you cloned for SPRING

**Comment by andrewvanbreda** [comment]
@DavidRoy @JimBacon 
I can tell you that SPRING uses the same calculation.
Jim: If you remember, could you let me know once you have done the fix please (I'll keep checking the other issue if you forget)

David: I corrected the map tool permission problem yesterday.
I think it would be good to have someone who has a detailed knowledge of what is supposed to happen check the SPRING pages.
It is very difficult for me to tell if things are working properly.
Thanks both

**Comment by JimBacon** [comment]
@andrewvanbreda  Fixed in https://github.com/Indicia-Team/client_helpers/commit/ca0f3b7a1210811bd8197e3b44e5b5190d6c66aa

**Comment by andrewvanbreda** [comment]
@JimBacon thanks Jim

**Comment by andrewvanbreda** [comment]
Hi @BirenRathod
I see I have done a commit to DEV, but you are in middle of Drupal upgrade.
Is it ok for me to deploy your commits to Test?, or alternatively deploy yours when you are ready, but could you let me know when you do that as my commit will be amongst them.
Cheers

**Comment by BirenRathod** [comment]
@andrewvanbreda Yes, I'm doing Drupal security release update, so please wait for my response.

**Comment by BirenRathod** [comment]
@andrewvanbreda I have finished upgrading site now.

**Comment by JimBacon** [comment]
@andrewvanbreda  I am reviewing my fix to the distance calculation as I'm not sure it is appropriate for some of the high latitude countries like Sweden.

**Comment by andrewvanbreda** [comment]
@JimBacon
Thanks Jim.
It is not an area or line of code I am familiar with. 
I will reverse the change on EU PoMS for now.
Let me know how you get on. Cheers

**Comment by JimBacon** [comment]
@andrewvanbreda I'm much happier with my revised solution. See https://github.com/Indicia-Team/client_helpers/commit/7478a45b85c1f70d6fc3f4c4c4e06352f6eb1ab4

**Comment by andrewvanbreda** [comment]
Hi @DavidRoy @JimBacon ,

This is now live on EU Pollinator.

I do have a couple of thoughts Jim.

1. I think the database will need correcting, but am not sure what you did to do that (I don't have access to the database anyway).
Are you able to help with that?
The website is the EU Pollinator Monitoring Scheme, and the locations types are Transect and Transect Section.
Having said that, there probably isn't any data that would have the problem if it is only happening very specifically.

2. I also noticed there are similar lines in other files.

ukbms_sectioned_transects_edit_transect.js has it, but going by the filename I assume that is only used on UK sites so perhaps that doesn't need the update

sectioned_transects_edit_transect.js also has a similar line but it uses different variable names. Line 665 in Master.
I am not even sure what site uses that, but I dunno if you think that file needs changing too?

Andy
